app/views/bill-runs/setup/no-licences.njk

```diff
@@ -3,27 +3,22 @@
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
 
 {% block breadcrumbs %}
-  {# Back link #}
   {{
     govukBackLink({
       text: 'Back',
-      href: '/system/bill-runs/setup/' + sessionId + '/region'
+      href: backlink
     })
   }}
 {% endblock %}
 
 {% block content %}
-  {# Main heading #}
-  <div class="govuk-body">
-    <h1 class="govuk-heading-l govuk-!-margin-bottom-3">{{ pageTitle }}</h1>
-  </div>
-
+  <h1 class="govuk-heading-l govuk-!-margin-bottom-3">{{ pageTitle }}</h1>
   {{ govukWarningText({
     text: 'Check there are licences ready to be billed and try again.',
     iconFallbackText: 'Warning'
   }) }}
 
-  <p class="govuk-body">
+  <p>
     <a href="/system/bill-runs" class="govuk-link">Return to bill runs</a>
```

Comment:
This doesn't need to be in a `<p>` as it's not text. if we need to add styling or padding we can add it as a class in the `<a>`